pulp_rpm/app/constants.py

```diff
@@ -13,11 +13,6 @@
 
 # The same as above, but in a format that choice fields can use
 CHECKSUM_CHOICES = (
-    (CHECKSUM_TYPES.UNKNOWN, CHECKSUM_TYPES.UNKNOWN),
```

Comment:
what was the uknown for?

Reply:
Pretty sure it shouldn't have been there in the first place.

Reply:
apparently it's even used https://github.com/pulp/pulp_rpm/blob/main/pulp_rpm/app/tasks/synchronizing.py#L159